lighthouse-core/gather/gather-runner.js

```diff
@@ -224,6 +282,13 @@ class GatherRunner {
 
     passes = this.instantiateGatherers(passes, options.config.configDir);
 
+    const gathererResults = {};
+    passes.forEach(pass => {
+      pass.gatherers.forEach(gatherer => {
+        gathererResults[gatherer.name] = [];
```

Comment:
thoughts on just making the beforePass initialize this? not really clear why you need to do this here